roles/preflight-checks/tasks/ceph.yml

```diff
@@ -26,3 +21,17 @@
   delegate_to: "{{ item }}"
   with_items:
     - "{{ groups['controller'] }}"
+
+- name: detect scale-out
```

Comment:
Can operator skip the tests using --skip-tags? Is it going to cause problem in such case? Does it need to be documented?